src/js/components/modals/EditAppModalComponent.jsx

```diff
@@ -73,7 +73,8 @@ var EditAppModalComponent = React.createClass({
 
   onRequestAppVersionError: function () {
     var props = this.props;
-    DialogActions.alert(`Could not fetch app '${props.appId}' version for editing: ${props.appVersion}`);
+    DialogActions.alert(`Could not fetch app '${props.appId}' version`
```

Comment:
Very good point. I tend to prefer boolean operators and concatenation on the same line, but ternary operators at the beginning of the statements.
So my preference would be

```
return doWeLikeTernaries
  ? <h1>YES</h1>
  : null;
```

and

```
var extraLongString = "dadadadadadadadadada" + 
  "dedede!";

// ... 
if (whateverAndEver &&
  alsoThisAndThat && 
  whyNotThose) {
```

Up for discussion. Open another issue?